### ISSUE TITLE
Using aria-hidden to remove content from the accessibility tree

### ISSUE THREAD

**Issue by kherr-a11y** [w3c/wcag#2041]
We seem to be missing a technique in the ARIA Techniques for Using aria-hidden to hide content from assistive technologies.  (https://www.w3.org/WAI/WCAG21/Techniques/#techniques).  This would be similar to H67 (https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H67)

**Comment by mbgower** [comment]
Thank you for this suggestion. I agree an `aria-hidden` technique would be sufficient to meet "Situation F: If the non-text content should be ignored by assistive technology". 

If you are interested in drafting such a technique, the Task Force would be happy to incorporate it.
